modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -274,13 +290,31 @@ $admin-bar-border: 1px solid var(--a-base-9);
 }
 
 .apos-admin-bar__context-title {
+  @include type-base;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
   flex: 1;
-  text-align: center;
+}
+
+.apos-admin-bar__context-title__icon {
+  display: inline-block;
+  margin-right: 5px;
+  line-height: 0;
 }
 
 .apos-admin-bar__context-controls {
+  display: flex;
+  align-items: center;
+  justify-content: end;
```

Comment:
`end` not doing it for chrome, `flex-end` aligns the buttons to the right side though